content/posts/blog/easy-xmpp-challenge.md

```diff
@@ -87,12 +86,39 @@ are a first step in that direction, and getting them into your client today
 will make new users' lives easier. But there is also potential to further
 streamline the whole process.
 
-### Call for Action
+### Easy Group Chats
+
+There are two typical IM use cases for group chats: chat with multiple friends
+at the same time (ad-hoc), or participate in a public chat room (typically for
+support purposes, pioneered by
+[Internet Relay Chat](https://en.wikipedia.org/wiki/Internet_Relay_Chat)).
+
+The latter is well-covered by [MUC](https://xmpp.org/extensions/xep-0045.html),
+and the upcoming [MIX](https://xmpp.org/extensions/xep-0369.html) should
+provide a solid technical basis for both use cases. The next logical step is
+to allow the easy
+[creation and sharing of ad-hoc rooms](https://wiki.xmpp.org/web/Easy_Group_Chats)
+between different clients, with a coherent user experience.
+
+### Fighting Spam
+
+If the amount of spam is a measure of XMPP's popularity, the network is doing
+exceptionally well. Almost all of today's XMPP spam can be blocked with some
+easy pattern matching rules, and the inevitable arms race will move to the
+next round.
+
+The community needs to prepare for that, by improving
+[reporting](https://xmpp.org/extensions/xep-0161.html) between server
+operators, adding [anti-spam
+features](https://xmpp.org/extensions/xep-0377.html) into clients and
+providing better whitelisting mechanisms to users.
+
+## Call for Action
 
-You can help making XMPP easier to use. As a user, you can contribute user
-stories, check your favorite client for confusing UI elements and non-helping
-wizards, and report those to the developers.  As a developer, you can
-streamline the on-boarding process in your client and contribute to the
+You can help making XMPP easier to use. As a user, you can contribute ideas,
+add user stories, check your favorite client for confusing UI elements and
```

Comment:
"'add user stories' where?" is a question I can see being asked. I suggest to make a few practical suggestions, such as:

* by reporting them in your clients support channel
* adding them to the Wiki (unlikely to happen since it requires a lot of action, like getting an account etc.)